### PR TITLE
Channels: Add support for multi-image community posts

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
closes https://github.com/iv-org/invidious/issues/3522

This implementation functions with JavaScript disabled and is fairly accessible
image
image

### CODE REVIEW

File: src/invidious/helpers/i18n.cr [hunk 1]
```diff
@@ -78,7 +78,7 @@ def load_all_locales
   return locales
 end
 
-def translate(locale : String?, key : String, text : String | Nil = nil) : String
+def translate(locale : String?, key : String, text : String | Nil = nil, texts : Hash(String, String) | Nil = nil) : String
```
Comment (SamantazFox) [inline src/invidious/helpers/i18n.cr:81]: You can do the following, and use `text.is_a?(Type)` to differentiate between the type of inputs:
```suggestion
def translate(locale : String?, key : String, text : String | Hash(String, String) | Nil = nil) : String
```

File: assets/css/carousel.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
+/*
```
Comment (SamantazFox) [inline assets/css/carousel.css:1]: Please fix trailing whitespaces in this file!

Comment (ChunkyProgrammer) [reply assets/css/carousel.css:1]: Done 🙂

**Comment by SamantazFox** [comment]
Thanks for your help ^^

**Pull request merged**